draft-ietf-rats-corim.md

```diff
@@ -996,6 +1004,58 @@ measurements for the Target Environment.
 {::include cddl/coswid-triple-record.cddl}
 ~~~
 
+#### Conditional Endorsement Series Triple {#sec-comid-triple-cond-series}
+
+A Conditional Endorsement Series triple uses a stateful environment, (i.e., `stateful-environment-record`),
+that identifies a Target Environment based on an `environment-map` plus the `measurement-map` measurements
+that have matching Evidence.
+
+The stateful Target Environment is a triple subject that MUST be satisfied before the series triple object is
+matched.
+
+~~~ cddl
+{::include cddl/stateful-environment-record.cddl}
+~~~
+
+The series object is an array of `conditional-series-record` that has both Reference and Endorsed Values.
+Each `conditional-series-record` record is evaluated in the order it appears in the series array.
+The Endorsed Values are accepted if the Reference Values in a `conditional-series-record` matches Evidence.
+The first `conditional-series-record` that sucessfully matches Evidence terminates the series and
+the matching Reference Values as well as the Endorsed Values are accepted.
+If none of the Reference Values in the series match Evidence, the triple is not matched,
+and no Claims are accepted.
+
+The `authorized-by` value in `measurement-map` in the stateful environment, if present,
+applies to all measurements in the triple, including `conditional-series-record` records.
+
+~~~ cddl
+{::include cddl/conditional-endorsement-series-triple-record.cddl}
+~~~
+
+~~~ cddl
+{::include cddl/conditional-series-record.cddl}
+~~~
+
+#### Conditional Endorsement Triple {#sec-comid-triple-cond-end}
+
+A Conditional Endorsement triple uses a stateful environment,`(i.e., stateful-environment-record`),
+that identifies a Target Environment based on an `environment-map` plus the `measurement-map` measurements
+that have matching Evidence.
+
+The stateful Target Environment is a triple subject that MUST be satisfied before the Endorsed Values in the
+triple object are accepted.
+
+~~~ cddl
+{::include cddl/stateful-environment-record.cddl}
+~~~
+
+The `authorized-by` value in `measurement-map` in the stateful environment, if present,
+applies to all measurements in the triple, including those in `measurement-values-map`.
```

Comment:
this does not apply here, right?

Reply:
Yes, because measurement-values-map measurements in the triple object are not qualified by a bounding `measurement-map` and because we said the scope of `authorized-by` is triple level scope. That implies the `authorized-by` in the `stateful-environment-record` applies to the triple object too.